Library/Formula/rbenv.rb

```diff
@@ -10,16 +10,37 @@ class Rbenv < Formula
   depends_on "ruby-build" => :recommended
 
   def install
-    inreplace "libexec/rbenv", "/usr/local", HOMEBREW_PREFIX
-    prefix.install Dir["*"]
+    inreplace "libexec/rbenv" do |s|
+      s.gsub!('"${BASH_SOURCE%/*}"/../libexec', libexec.to_s, false)
```

Comment:
Without the `false` it wouldn't just warn, it was fatal. This was for 0.4.0 compatibility. However you're right that this doesn't make sense anymore now that the formula is bumped to 1.0.0.